### ISSUE TITLE
Missing `description` attribute using ensembles.change_significance

### ISSUE THREAD

**Issue by tlogan2000** [Ouranosinc/xclim#1728]
### Setup Information

- Xclim version: 0.47.0
- Python version: 3.10
- Operating System: Linux

### Description

Using version 0.47.0 with following code
```
from xclim import ensembles as xens
chng_f, pos_f = xens.change_significance(
    fut,ref),test="ttest",
)
```
`pos_f` output no longer seems to have any attributes
using `xclim`  0.43.0  attrs included

`description :
    Fraction of members showing significant change that agree on a positive change. Significant change was tested with test ttest with parameters p_change: 0.05.
units :
test :
    ttest
history :
    fut: 2011-06-01T01:08:07Z CMOR rewrote data to comply with CF standards and CMIP5 requirements.
    2016-01-18T18:02:53: Interpolate to nrcan_livneh grid.
    2016-02-10T09:50:11: Bias correction using nrcan_livneh.
    ref: 2011-06-01T01:08:07Z CMOR rewrote data to comply with CF standards and CMIP5 requirements.
    2016-01-18T18:02:53: Interpolate to nrcan_livneh grid.
    2016-02-10T09:50:11: Bias correction using nrcan_livneh.
    [2024-04-23 14:44:10] : pos_frac from change_significance(fut=fut, ref=ref, test=ttest, p_change: 0.05) - xclim version: 0.43.0`

### Contribution

- [ ] I would be willing/able to open a Pull Request to address this bug.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by aulemahal** [comment]
Hé, that's because `change_significance` is deprecated. Can you use `robust_fractions` instead ?

**Comment by Zeitsperre** [comment]
Closed via #1737
